conda/common/disk.py

```diff
@@ -207,11 +195,23 @@ def rm_rf(path, max_retries=5, trash=True):
                 if not isdir(path):
                     from conda.install import delete_linked_data_any
                     delete_linked_data_any(path)
+        elif lexists(path):
```

Comment:
This block of code is identical to above. I just rearranged the order of flow control to check `isdir` first and then use `elif lexists`.